openreview/conference/invitation.py

```diff
@@ -416,9 +448,8 @@ def __init__(self, conference, note, reveal_authors, reveal_submission, email_pc
 
             super(PaperWithdrawInvitation, self).__init__(
                 id=conference.get_invitation_id('Withdraw', note.number),
+                super=conference.get_invitation_id('Paper_Withdraw'),
```

Comment:
given the invitation is created using a super invitation, all the properties that are defined in the super invitation don't need to be defined in the child invitation, for example `content`, you can remove it.